backend/src/v1/services/external-consumer-service.ts

```diff
@@ -120,72 +145,133 @@ const externalConsumerService = {
       );
     }
 
+    /**
+     * 1) Create a union of the pay_transparency_report and report_history table as reports
+     * 2) Create a union of the pay_transparency_calculated_data and calculated_data_history as calculated
+     * 3) Paginate the reports
+     * 4) Join reports and calculated_data based on report_change_id
+     */
+    const getReportsQuery = Prisma.sql`select *
+     from ((select report.report_id,
+                  report.report_id as report_change_id,
+                  report.company_id,
+                  report.user_id,
+                  report.user_comment,
+                  report.employee_count_range_id,
+                  report.naics_code,
+                  report.report_start_date,
+                  report.report_end_date,
+                  report.create_date,
+                  report.update_date,
+                  report.create_user,
+                  report.update_user,
+                  report.report_status,
+                  report.revision,
+                  report.data_constraints,
+                  report.is_unlocked,
+                  report.reporting_year,
+                  report.report_unlock_date,
+                  naics_code.naics_label,
+                  company.company_id,
+                  company.company_name,
+                  company.bceid_business_guid,
+                  company.address_line1,
+                  company.address_line2,
+                  company.city,
+                  company.province,
+                  company.country,
+                  company.postal_code,
+                  employee_count_range.employee_count_range
+           from pay_transparency_report as report
+		       left join pay_transparency.naics_code as naics_code on naics_code.naics_code = report.naics_code
```

Comment:
Can you remove "pay_transparency." from all the table names? I think this was my fault. Added them in to do my own test, but forgot to remove them when I sent the draft to you. I count 10 instances of "pay_transparency." in the query